tools/fork-network/src/cli.rs

```diff
@@ -543,13 +555,61 @@ impl ForkNetworkCommand {
         Ok(())
     }
 
+    // Read the values that used to be written before the changes that have us write to FORK_TOOL_FLAT_HEAD
+    // and FORK_TOOL_SHARD_LAYOUT
+    fn legacy_get_state_roots_and_hash(
+        &self,
+        store: Store,
+        epoch_manager: &dyn EpochManagerAdapter,
+    ) -> anyhow::Result<(HashMap<ShardUId, StateRoot>, BlockInfo, EpochId, ShardLayout)> {
+        let epoch_id = EpochId(store.get_ser(DBCol::Misc, b"FORK_TOOL_EPOCH_ID")?.unwrap());
```

Comment:
nit: These three keys (`b"FORK_TOOL_EPOCH_ID"`, etc) now appear multiple times, we could use a constant

Reply:
done